sources/ClangSharp/Cursors/Decls/UsingDirectiveDecl.cs

```diff
@@ -0,0 +1,13 @@
+using ClangSharp.Interop;
+
+namespace ClangSharp
+{
+    // e.g. `using namespace std;`
+    // See: https://clang.llvm.org/doxygen/classclang_1_1UsingDirectiveDecl.html
+    public class UsingDirectiveDecl: NamedDecl, IMergeable<UsingDecl>
```

Comment:
nit: There should be a space before the colon

Reply:
Feel free to insert it.